hub/src/PeerController.cpp

```diff
@@ -39,6 +39,72 @@ void CPeerController::refresh() {
     });
 }
 
+const QString& CPeerController::status_description(const QString &status){
+    static std::map<QString, QString> dct_desp = {
+
+        {"aborted", tr ("The Peer is in an aborted state. This means that it was abruptly "
+                        "stopped without properly closing the session. Run `start` "
+                        "to resume this virtual machine. If any problems persist, you may "
+                        "have to destroy and restart the virtual machine.")},
+
+        {"gurumeditation", tr ("The Peer is in the \"guru meditation\" state. This is a rare case which means "
+                               "that an internal error in VirtualBox caused the Peer to fail. This is always "
+                               "the sign of a bug in VirtualBox. You can try to bring your VM back online "
+                               "with a `start`.")},
```

Comment:
@Erkin97 "The Peer was failed because of internal error (or bug) in VirtualBox. You can try to bring your VM back online with `start`."